samples/KeyValues/gameinfo.vdf

```diff
@@ -0,0 +1,17 @@
+#base "default"
+"GameInfo"
+{
+	game		"Example"
+	title		"Example Game"
+	gamelogo	1 // true
+	FileSystem
+	{
+		SteamAppId	100
+		SearchPaths 	// search for game paths
+		{
+			Game	|default_path|
+			Game	|default_path|../game
+			Game	example
+		}
+	}
+}
```

Comment:
🤔 this is the same as the gameinfo.txt file. Is this intended?

Also, are these really real world examples? They look like "hello world"-like to me.

Reply:
Switched out the gameinfo.txt.
The vdf one is what I used to test the syntax highlighter and includes more language features than any of the real-world examples which are all either tiny or just `key val key {key val key val}` etc with nothing more.